draft-ietf-acme-acme.md

```diff
@@ -1759,11 +1765,13 @@ Revocation requests are different from other ACME request in that they can be
 signed either with an account key pair or the key pair in the certificate.
 Before revoking a certificate, the server MUST verify that the key used to sign
 the request is authorized to revoke the certificate.  The server SHOULD consider
-at least the following keys authorized for a given certificate:
+at least the following accounts authorized for a given certificate:
```

Comment:
Problem with this is that the public key in the certificate might not have an account.  You could refactor this to say "The following accounts: [issued, authorized]  The server SHOULD also consider a revocation request valid if it is signed with the private key corresponding to the public key in the certificate."